externals/llvm-external-projects/torch-mlir-dialects/include/torch-mlir-dialects/Dialect/Tcp/IR/TcpBase.td

```diff
@@ -0,0 +1,43 @@
+//===-------------------------------------------------------*- tablegen -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// Also available under a BSD-style license. See LICENSE.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef TORCH_MLIR_DIALECT_TCP_BASE
+#define TORCH_MLIR_DIALECT_TCP_BASE
+
+include "mlir/IR/OpBase.td"
+
+def Tcp_Dialect : Dialect {
+  let name = "tcp";
+  let cppNamespace = "::mlir::torch::tcp";
```

Comment:
`::mlir::tcp` perhaps, given the plans to be applicable more widely than to PyTorch?

Reply:
Given that we are bootstrapping TCP under TorchMLIR externals, I assumed it has to use the `::mlir::torch::tcp` namespace. 

@silvasean Is it okay to use `::mlir::tcp` for this instead?

Reply:
Given the plans to make it more widely applicable, mlir::tcp seems fine.

Reply:
Thanks for clarifying. Updated it to `mlir::tcp`